packages/tocco-util/src/form/formBuilder.js

```diff
@@ -42,6 +43,11 @@ export default (
   }
 
   const shouldRenderField = (formDefinitionField, entityField) => {
+    if (mode && formDefinitionField.scopes && formDefinitionField.scopes.length > 0
```

Comment:
I think the check `formDefinitionField.scopes.length > 0` can be removed.

Reply:
why? by default scopes as returned by REST is an empty array. If no scope set (array empty) we dont want to regard the current mode (create or update) and display the field anyways.